lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -90,6 +94,25 @@ def valid_attestation_statement?
       @attestation_type, @attestation_trust_path = attestation_statement.valid?(authenticator_data, client_data.hash)
     end
 
+    def valid_attestation_trustworthiness?
+      case @attestation_type
+      when WebAuthn::AttestationStatement::ATTESTATION_TYPE_NONE
+        WebAuthn.configuration.acceptable_attestation_types.include?(:None)
+      when WebAuthn::AttestationStatement::ATTESTATION_TYPE_SELF
+        WebAuthn.configuration.acceptable_attestation_types.include?(:Self)
+      else
+        return false unless WebAuthn.configuration.acceptable_attestation_types.include?(:Basic)
+
+        trust_store = OpenSSL::X509::Store.new.tap do |store|
+          attestation_root_certificates_store.each do |cert|
```

Comment:
Why are we building the trust store here? Based on its name `attestation_root_certificates_store` below in this file should return one.